rust/Dockerfile

```diff
@@ -73,11 +73,20 @@ ENV PACKAGE=${PACKAGE}
 
 CMD $PACKAGE
 
+FROM node:20-alpine AS node-utils
+ENV PNPM_HOME="/pnpm"
+ENV PATH="$PNPM_HOME:$PATH"
+RUN corepack enable
+COPY docker-browser-tests /docker-browser-tests
+WORKDIR /docker-browser-tests
+RUN pnpm install
+
```

Comment:
I think it's cleaner if you keep our Rust Dockerfile free of this and just extend the `debug` image with another Dockerfile inside `scripts/tests`

That will prevent someone else from accidentally hitting the same hangs you did